install_template/templates/products/edb-postgres-advanced-server/sles-12.njk

```diff
@@ -1,6 +1,11 @@
 {% extends "products/edb-postgres-advanced-server/base.njk" %}
 {% set platformBaseTemplate = "sles-12" %}
-{% block postinstall %}
-{% include "products/edb-postgres-advanced-server/_centos-rhel-postinstall.njk" %}
-{{ super() }}
-{% endblock postinstall %}
+{% block individual_component %}
+To install an individual component:
+
+```shell
+zypper install <package_name>
+```
+
+Where `package_name` can be any of the available package from the [available package list](https://www.enterprisedb.com/docs/epas/latest/epas_inst_linux/install_details/rpm_packages/).
```

Comment:
Try to use relative paths for links within docs, e.g. `/epas/latest/epas_inst_linux/install_details/rpm_packages/`